### PR TITLE
Color assets + extension to call them + Rootstrap color palette

### PR DESCRIPTION
**Pull request by pMalvasio**
#### Description

* Color assets
* Extension to call them
* Rootstrap color palette anticipating dark mode (next PR)

---

#### Preview

Screen Shot 2020-07-30 at 5 31 25 PM

### CODE REVIEW

File: swift-ui-base/Extensions/ColorExtension.swift [hunk 1]
```diff
@@ -7,27 +7,12 @@
...
-  static let errorRed = UIColor(red: 255, green: 81, blue: 82).asColor()
-}
-
-extension UIColor {
-  convenience init(red: Int, green: Int, blue: Int) {
-    self.init(red: min(CGFloat(red), 255.0) / 255.0,
-              green: min(CGFloat(green), 255.0) / 255.0,
-              blue: min(CGFloat(blue), 255.0) / 255.0,
-              alpha: 1.0)
-  }
```
Comment (glm4) [inline swift-ui-base/Extensions/ColorExtension.swift:28]: Why removing this handy initializer?

**Review by glm4 — APPROVED** [review]

**Review by CamilaMoscatelli — APPROVED** [review]

File: swift-ui-base/Extensions/ColorExtension.swift (hunk 1 above)
Comment (pMalvasio) [reply swift-ui-base/Extensions/ColorExtension.swift:28]: The idea is to create all as a color asset, that means that you can pass the `0.34234` RGB value directly to the asset. the initializer should not be used again.

Comment (pMalvasio) [reply swift-ui-base/Extensions/ColorExtension.swift:28]: You can select `Floating point` there and that's it, only once to create the asset and you're ready to go. And actually the default selection on that dropdown menu is that one

Screen Shot 2020-07-30 at 5 38 00 PM

**Review by germanStabile — APPROVED** [review]

File: swift-ui-base/Extensions/ColorExtension.swift (hunk 1 above)
Comment (pMalvasio) [reply swift-ui-base/Extensions/ColorExtension.swift:28]: Also, `Color(red: , green: , blue: )` for values `0-255` already exists as part of the swiftUI core

**Pull request merged**